src/query/execute_schema.c

```diff
@@ -2248,6 +2250,71 @@ do_alter_user (const PARSER_CONTEXT * parser, const PT_NODE * statement)
 	}
     }
 
+  /* member */
+  node = statement->info.alter_user.members;
+  member_name = (node && IS_NAME (node)) ? GET_NAME (node) : NULL;
+  if (member_name != NULL)
+    {
+      if (alter_user_code == PT_ADD_MEMBERS && (ws_is_same_object (user, Au_user) || au_is_dba_group_member (Au_user)))
```

Comment:
How about checking ws_is_same_object and au_is_dba_group_member before checking alter_user_code?

Reply:
We've fixed it, please confirm

Reply:
I thought of separating the if statements checking `ws_is_same_object` and `au_is_dba_group_member`.

Reply:
The code you reviewed looks more readable, so we've fixed it.